<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LV-IND-LPT-226" ignored="4" name="Test" tests="7" failures="5" timestamp="24 Jun 2019 11:13:27 GMT" time="79.475" errors="0">
  <testcase name="testvalidLogin" time="11.58" classname="com.Zoho.TestCases.InvalidTestCase">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.Zoho.TestCases.InvalidTestCase.testvalidLogin(InvalidTestCase.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testvalidLogin -->
  <testcase name="testvalidLogin" time="11.671" classname="com.Zoho.TestCases.InvalidTestCase">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.Zoho.TestCases.InvalidTestCase.testvalidLogin(InvalidTestCase.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testvalidLogin -->
  <testcase name="testvalidLogin" time="11.425" classname="com.Zoho.TestCases.InvalidTestCase">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.Zoho.TestCases.InvalidTestCase.testvalidLogin(InvalidTestCase.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testvalidLogin -->
  <testcase name="testvalidLogin" time="11.453" classname="com.Zoho.TestCases.InvalidTestCase">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.Zoho.TestCases.InvalidTestCase.testvalidLogin(InvalidTestCase.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testvalidLogin -->
  <testcase name="testvalidLogin" time="11.38" classname="com.Zoho.TestCases.InvalidTestCase">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.Zoho.TestCases.InvalidTestCase.testvalidLogin(InvalidTestCase.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testvalidLogin -->
  <testcase name="testvalidLogin" time="10.402" classname="com.Zoho.TestCases.TestValidLogin"/>
  <testcase name="Validation" time="10.095" classname="com.Zoho.TestCases.zohoHomeTest"/>
  <testcase name="@BeforeSuite Setup" time="0.0" classname="com.Zoho.TestCases.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite Setup -->
  <testcase name="@BeforeSuite Setup" time="0.0" classname="com.Zoho.TestCases.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite Setup -->
  <testcase name="@AfterSuite tearDown" time="0.0" classname="com.Zoho.TestCases.BaseClass">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase name="@AfterSuite tearDown" time="0.0" classname="com.Zoho.TestCases.BaseClass">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
</testsuite> <!-- Test -->
